config.toml

```
@@ -35,294 +35,190 @@ time_format_blog = "2006-01-02"
+Declarative management using e.g. [YAML](https://yaml.org) and [Git](https://git-scm.com/) is prominent in the cloud native space, but not
+equally in the embedded and firmware space. We believe declarative APIs are very useful and powerful,
+as they allow for observe-diff-act reconciliation loops. Consistent Kubernetes-style APIs will be used
+across the stack.
 """
 
 [[params.overview.features]]
-icon   = "fas fa-code fa-3x"
-header = "Auditable"
+icon   = "fas fa-drafting-compass fa-3x"
+header = "Open Source"
 text   = """
-Everything is controlled through pull requests. Your Git history provides a sequence of transactions, allowing you to
-recover state from any snapshot.
+Racklet is 100% open source software and hardware. Anyone can contribute, improve,
+fork and access the project. We want to bring innovation forward e.g. by publishing and
+using open (source) APIs, sharing code freely, and collaborating with fellow community members. 
 """
 
 [[params.overview.features]]
-icon   = "fas fa-drafting-compass fa-3x"
-header = "Designed for Kubernetes"
+icon   = "fas fa-robot fa-3x"
+header = "Accessible"
 text   = """
-Declaratively configurable using Custom Resources, state reports in the
-[object's status](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#object-spec-and-status)
-and via [Kubernetes Events](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application-introspection/),
-and integrations with Kubernetes RBAC.
+Only use commonly available components that can be aquired in most parts of the world in a frictionless
+manner. For non-off-the-shelf casing and PCBs, we provide 3D-printable designs and easily reproducible
+schematics, respectively.
 """
 
-
 [[params.overview.features]]
 icon   = "fas fa-box-open fa-3x"
-header = "Out-of-the-box integrations"
+header = "Upgradable"
 text   = """
-Support for e.g. [Kustomize](https://kustomize.io), [Helm](https://helm.sh); GitHub, GitLab, Harbor and custom
-webhooks; notifications to most team communication platforms; and many more.
+The modularity of the system should allow that individual pieces of the system (e.g. compute,
+storage, network switches, power supplies) should be upgradable without having to disrupt the
+rest of the rack, or disregard existing, functioning parts. This will minimize E-waste produced over time.
 """
 
 [[params.overview.features]]
 icon   = "fas fa-cube fa-3x"
 header = "Extensible"
 text   = """
-Easily create a continous delivery solution with only the components you need, or use the [GitOps Toolkit](#gitops-toolkit)
-to extend Flux.
+The hardware, software, and APIs all strive to be as modular and extensible as possible.
+We want to follow the Unix philosophy to allow for portability between e.g. different hardware
+modules implementing the same interfaces, or extensibility where the user demands custom features.
```

```suggestion
modules implementing the same interfaces, or extensibility where the user demands customized features.
```
"custom" is an informal abbreviation, it's better to always write it out.